java/flight/flight-jdbc-driver/pom.xml

```diff
@@ -147,6 +137,28 @@
             <artifactId>flight-sql</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.calcite.avatica</groupId>
+            <artifactId>avatica</artifactId>
+            <version>1.18.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
```

Comment:
Are we sure this dependency doesn't show up elsewhere? If it does, let's put it in dependency management.

Reply:
It isn't used. All dependencies were added here are just because of flight-jdbc-driver.